Source/SPFavoriteColorSupport.m

```diff
@@ -80,6 +80,18 @@ - (NSColor *)colorForIndex:(NSInteger)colorIndex
 
 - (NSArray *)userColorList
 {
+	if (@available(macOS 10.13, *)) {
```

Comment:
There should be macro for these checks

Reply:
Ah right, you mean the stuff defined in SPCompatibility.h ? I missed that before